src/panel-triggers/components/AlertList/AlertList.tsx

```diff
@@ -161,9 +175,23 @@ export default class AlertList extends PureComponent<AlertListProps, AlertListSt
     return amount;
   }
 
+  getAmountOfAlertsWithTestTag = (problems: any[]) => {
+    let amount = 0;
+    problems.forEach((problem: any) => {
+      if (problem.tags) {
+        problem.tags.forEach((tag: any) => {
+          if (tag.tag === 'test') {
+            amount++;
```

Comment:
Can a trigger have multiple `test` tags? (At least in the future items can have multiple `application` tags) The inner loop could be for instead of forEach and it could break if a test tag is found to avoid calculating same problem twice.

Reply:
SonarLint recommended a new style syntax for the for-loop:
```
for (let tag of problem.tags) {
 ...
}
```
I have never seen this kind of syntax :)